dags/deletes.py

```diff
@@ -174,26 +183,18 @@ def load_pending_person_deletions(
 
     if not create_pending_deletes_table.team_id:
         # Use Django's queryset iterator for memory efficiency
-        pending_deletions = (
-            AsyncDeletion.objects.filter(
-                deletion_type=DeletionType.Person,
-                delete_verified_at__isnull=True,
-                created_at__lte=create_pending_deletes_table.timestamp,
-            )
-            .values("team_id", "key", "created_at")
-            .iterator()
-        )
+        pending_deletions = AsyncDeletion.objects.filter(
+            deletion_type=DeletionType.Person,
+            delete_verified_at__isnull=True,
+            created_at__lte=create_pending_deletes_table.timestamp,
+        ).iterator()
     else:
-        pending_deletions = (
-            AsyncDeletion.objects.filter(
-                deletion_type=DeletionType.Person,
-                team_id=create_pending_deletes_table.team_id,
-                delete_verified_at__isnull=True,
-                created_at__lte=create_pending_deletes_table.timestamp,
-            )
-            .values("team_id", "key", "created_at")
-            .iterator()
-        )
+        pending_deletions = AsyncDeletion.objects.filter(
+            deletion_type=DeletionType.Person,
+            team_id=create_pending_deletes_table.team_id,
+            delete_verified_at__isnull=True,
+            created_at__lte=create_pending_deletes_table.timestamp,
+        ).iterator()
```

Comment:
These objects are lazily evaluated, so you could do something like this if you wanted to reduce duplication:

```python
pending_deletions = AsyncDeletion.objects.filter(
    deletion_type=DeletionType.Person,
    delete_verified_at__isnull=True,
    created_at__lte=create_pending_deletes_table.timestamp,
)

if create_pending_deletes_table.team_id:
     pending_deletions = pending_deletions.filter(team_id=create_pending_deletes_table.team_id)
```